### PR TITLE
Add labels for github issue templates

### LINKED ISSUE #44905: Github BUG template disappeared
The normal bug template disappeared from the issue form:
image

### PR DESCRIPTION
**Pull request by phofl**
- [x] closes #44905

Looks like they changed the format a bit

You can check how they look under https://github.com/phofl/pandas/issues/new/choose

cc @twoertwein

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/installation_issue.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ body:
   - type: checkboxes
     id: checks
     attributes:
+      label: Pandas version checks
```
Comment (mroeschke) [inline .github/ISSUE_TEMPLATE/installation_issue.yaml:10]: Nit: Maybe `Installation check`?

Comment (phofl) [reply .github/ISSUE_TEMPLATE/installation_issue.yaml]: Thx, missed that

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks for the quick resolution. Looks like the issue templates are working again 👍

**Comment by phofl** [comment]
Great :)

**Pull request merged**